app/views/api/v1/posts/show.json.jbuilder

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+json.call(@post, :id, :title, :body, :published_at, :user_id)
+
+json.likes @post.likes do |like|
+  json.user_id like.user_id
+  user = User.find(like.user_id)
+  json.nickname user.nickname
+end
+
+json.comments @post.comments do |comment|
+  json.extract! comment, :id, :content
+
+  json.replies comment.comments do |reply|
```

Comment:
This finishes in the first level of replies. If you also have a reply for that reply, it won't be shown. Let's make this recursive using partials!